libs/common-traits/src/lib.rs

```diff
@@ -278,3 +278,183 @@ impl<T> PreConditions<T> for Never {
 pub trait TrancheToken<PoolId, TrancheId, CurrencyId> {
 	fn tranche_token(pool: PoolId, tranche: TrancheId) -> CurrencyId;
 }
+
+pub mod fees {
+	use frame_support::dispatch::DispatchResult;
+	use frame_support::traits::tokens::Balance;
+
+	/// A way to identify a fee value.
+	pub enum Fee<Balance, FeeKey> {
+		/// The fee value itself.
+		Balance(Balance),
```

Comment:
I think once we have standardized the usage of the fees pallet across all consumer pallets, the balance fee might not be needed, since the fee value should be gathered from the pallet_fees storage.

Reply:
There is still some cases, as in the commit usage, where the fee is extracted from the storage, multiplied and then pay to the author where the Balance variant is needed today.

Other possible approach I have been evaluating is to add an optional map function argument to mutate the value before make the payment. In this way we force to use the fee storage for anybody who wants to pay something, but it's still flexible to change the value if needed.

Reply:
ah right! the multiplier part of the commit fee 👍  I think is fine then, maybe later we can have a more dynamic approach like you suggests.